metadata-ingestion/tests/integration/lookml/expected_output.json

```diff
@@ -1434,5 +1434,219 @@
         "lastObserved": 1586847600000,
         "runId": "lookml-test"
     }
+},
+{
+    "proposedSnapshot": {
+        "com.linkedin.pegasus2avro.metadata.snapshot.DatasetSnapshot": {
+            "urn": "urn:li:dataset:(urn:li:dataPlatform:looker,lkml_samples.view.view_derived_explore,PROD)",
+            "aspects": [
+                {
+                    "com.linkedin.pegasus2avro.common.BrowsePaths": {
+                        "paths": [
+                            "/prod/looker/lkml_samples/views"
+                        ]
+                    }
+                },
+                {
+                    "com.linkedin.pegasus2avro.common.Status": {
+                        "removed": false
+                    }
+                },
+                {
+                    "com.linkedin.pegasus2avro.dataset.UpstreamLineage": {
+                        "upstreams": [
+                            {
+                                "auditStamp": {
+                                    "time": 0,
+                                    "actor": "urn:li:corpuser:unknown"
+                                },
+                                "dataset": "urn:li:dataset:(urn:li:dataPlatform:looker,data.explore.my_view_explore,PROD)",
+                                "type": "VIEW"
+                            }
+                        ],
+                        "fineGrainedLineages": [
+                            {
+                                "upstreamType": "FIELD_SET",
+                                "upstreams": [
+                                    "urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:looker,data.explore.my_view_explore,PROD),my_view_explore.country)"
```

Comment:
I think, the correct schema field urns would be 

"urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:looker,data.explore.my_view_explore,PROD),country)"

Probably need to strip the  starting explore name from column name ?

Reply:
actually this is correct - explore fields have the explore name as part of the schema

Reply:
Oh, You mean this - 
https://github.com/datahub-project/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/schema/SchemaMetadataKey.pdl#L17 

https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/tests/integration/looker/golden_looker_mces.json#L141

I wasn't aware that is is considered when generating schema field urns.

Reply:
yup, although this is only applicable for looker explores

Reply:
Ah, okay. Curious, if we have tested this on UI, that column lineage edge shows up correctly, or is if any change required there. Otherwise looks good.

Reply:
yes I tested it locally